### PR TITLE
bootstrap play 2.8 upgrade

### PR DESCRIPTION
**Pull request by kuldeepwadhwa9**

### CODE REVIEW

File: test/unit/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResultSpec.scala [hunk 1]
```diff
@@ -41,10 +41,6 @@ class ScanResultSpec extends UnitSpec {
       )
     )
 
-    "Convert Result to JSON" in {
```
Comment (anburoach) [inline test/unit/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResultSpec.scala:44]: May I know.. why did you remove this test?

File: test/unit/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResultSpec.scala [hunk 2]
```diff
@@ -65,10 +61,6 @@ class ScanResultSpec extends UnitSpec {
...
```
Comment (anburoach) [inline test/unit/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResultSpec.scala:68]: same as above

File: test/unit/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResultSpec.scala (hunk 1 above)
Comment (kuldeepwadhwa9) [reply test/unit/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResultSpec.scala:44]: I've reverted this test

File: test/unit/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResultSpec.scala (hunk 2 above)
Comment (kuldeepwadhwa9) [reply test/unit/uk/gov/hmrc/bindingtarifffilestore/model/upscan/ScanResultSpec.scala:68]: I've reverted this test

**Review by anburoach — APPROVED** [review]
Approved
